astropy/time/core.py

```diff
@@ -1771,6 +1773,49 @@ def light_travel_time(self, skycoord, kind='barycentric', location=None, ephemer
         tcor_val = (spos * cpos).sum(axis=-1) / const.c
         return TimeDelta(tcor_val, scale='tdb')
 
+    def earth_rotation_angle(self, longitude=None):
+        """Calculate local Earth rotation angle.
+
+        Parameters
+        ---------------
+        longitude : `~astropy.units.Quantity`, `~astropy.coordinates.EarthLocation`, str, or None; optional
+            The longitude on the Earth at which to compute the Earth rotation
+            angle (taken from a location as needed).  If `None` (default), taken
+            from the ``location`` attribute of the Time instance. If the special
+            string 'tio', the result will be relative to the Terrestrial
+            Intermediate Origin (i.e., the output of `~erfa.era00`).
+
+        Returns
+        -------
+        `~astropy.coordinates.Longitude`
+            Local Earth rotation angle with units of hourangle.
+
+        See Also
+        --------
+        astropy.time.Time.sidereal_time
+
+        Notes
+        -----
+        The difference between apparent sidereal time and Earth rotation angle
+        is the equation of the origins which is the angle between the CIO and
+        the equinox. Applying apparent sidereal time to the hour angle yields
+        the true apparent Right Ascension with respect to the equinox, while
+        applying the Earth rotation angle yields the intermediate Right
```

Comment:
What does `intermediate` actually mean in this context? Is that just the relative RA from the CIO?

Reply:
Please see the IAU 2006 glossary (link above).